File: Service_eHospital_DongNai_A_NSTL/Implement/NS_DeNghiKhenThuongService.cs
namespace Service_eHospital_DongNai_A_NSTL.Implement
{
    public class NS_DeNghiKhenThuongService : BaseService<NS_DeNghiKhenThuong, INS_DeNghiKhenThuongRepository>
    , INS_DeNghiKhenThuongService
    {
    private readonly INS_DeNghiKhenThuongRepository _NS_DeNghiKhenThuongRepository;
    public NS_DeNghiKhenThuongService(INS_DeNghiKhenThuongRepository NS_DeNghiKhenThuongRepository) : base(NS_DeNghiKhenThuongRepository)
    {
    _NS_DeNghiKhenThuongRepository = NS_DeNghiKhenThuongRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A_NSTL/NS_DeNghiKhenThuong.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NS_DeNghiKhenThuong } from './NS_DeNghiKhenThuong.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NS_DeNghiKhenThuongService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'Thang', 'Nam', 'MaNhanVien', 'GhiChu'];

    List: NS_DeNghiKhenThuong[] | undefined;
    ListFilter: NS_DeNghiKhenThuong[] | undefined;
    FormData!: NS_DeNghiKhenThuong;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NS_DeNghiKhenThuong";
    }
}

